lib/containers/urls.pm

```diff
@@ -262,12 +263,11 @@ sub supports_image_arch {
     (grep { $_ eq $arch } @{$images_uri{$distri}{$version}{available_arch}}) ? 1 : 0;
 }
 
-# Returns a tuple of image urls.
-# If empty, no images available.
-sub get_urls_from_var {
+# Returns an array with the contents of the given variable or undef, if the var is not set
+sub get_container_url_from_var {
     my $var = shift;
-    my @urls = split(/,/, trim(get_var($var)));
-    return (\@urls);
+    my @urls = (trim(get_var($var)));
+    return (length(@urls[0]) > 0) ? (\@urls) : undef;
```

Comment:
`s/length(@urls[0])/$urls[0]/` should do the same, and it is more perlist way

Reply:
Done.